common/workunit/workunit.cpp

```diff
@@ -3027,6 +3028,7 @@ class CWorkUnitFactory : public CInterface, implements IWorkUnitFactory, impleme
                 sortElements(iter, sortOrder.get(), NULL, NULL, unknownAttributes, elements);
                 return conn.getClear();
             }
+            virtual unsigned getTotalElements() { throw MakeStringException(-1, "not implemented"); }
```

Comment:
Based on the initial comment 'the result returned here should still be adjusted by the postfilter as it was before.', the 'total' should be elements.ordinality() - postfilters.